make-checkout

```diff
@@ -67,6 +68,14 @@ def main():
     execute("git", "fetch", "origin", opts.ref, cwd=TARGET_DIR, error_on_fail=False)
     execute("git", "checkout", "--detach", opts.revision, cwd=TARGET_DIR, error_on_fail=False)
 
+    if opts.base:
```

Comment:
And this needs to be `opts.branch` with the current PR, but it should better be `opts.rebase` wrt. my comment above.

Reply:
noticed when I ran it :) But thanks for double checking after me ;)